components/message-composer.jsx

```diff
@@ -52,6 +54,8 @@ export default class MessageComposer extends PureComponent {
             value: '',
             busy: true
         });
+        this.props.client.stopPeeking();
+        await this.props.client.joinRoom(this.props.roomId, {syncRoom: true});
```

Comment:
> ![Screenshot 2020-06-28 at 12 57 24](https://user-images.githubusercontent.com/1197991/85946856-30bd1e00-b93f-11ea-8626-81858504c110.png)
> 
> Interesting bug - the message is duplicated in the client it was sent from. Refreshing cleared the problem, and I could not reproduce.

@benparsons 
I'm not completely sure why this happens. Since the client is running in guest mode here, whenever someone attempts to leave a comment I call `stopPeeking` and then `joinRoom` just to make sure they're actually a member of the room before sending the event. The issue only presents itself when they actually weren't a member before and end up joining the room. It's probably got something to do with this. 
Either the wrong name appears on the first message (the user ID instead of the name, but is correct for subsequent messages) or the messages appear to be duplicated. These only show up in the client the message was sent from and seem to disappear on refreshing.